tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/dashboard/documentdashboard.controller.js

```diff
@@ -286,8 +299,9 @@
                     vm.isDisplayMessage = false;
                 }
             }
+            //#endregion
 
-            //Event is going to fire when the user clicks on "Email as attachment" or "Email as link" in the modal window
+            //#region Event is going to fire when the user clicks on "Email as attachment" or "Email as link" in the modal window
             vm.downloadEmailAsAttachment = function (downloadAttachmentsAsEmail) {

```

Comment:
elaborate more on this. why we need this event to fire on clicking on email as attachment?